src/aat/resources/features/F-102/S-534.td.json

```diff
@@ -0,0 +1,37 @@
+{
+  "_guid_": "S-534",
+  "_extends_": "F-102_Test_Data_Base",
+  "title": "must return 404 for if no jurisdiction is found",
+  "specs": [
+    "a detailed profile in CCD having create case access for a jurisdiction",
+    "contains HTTP 400 Bad Request",
+    "has DELETE as case access parameter",
+    "contains an error message : Access can only be 'create', 'read' or 'update'"
+  ],
+  "request": {
+    "queryParams": {
+      "access": "delete"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "_extends_": "Common_400_Response_Headers",
+      "Content-Length": "[[ANY]]",
+      "Transfer-Encoding": "[[ANY]]",
+      "Connection": "[[ANY]]",
+      "Content-Type": "[[ANY]]",
+      "Content-Encoding": "[[ANY]]"
+    },
```

Comment:
Headers data can move to F-102_Test_Data_Base.td.json to make reusability.

Reply:
Please update this to other scenarios - test data files.

Reply:
Updated the code.